src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -11,33 +11,33 @@ namespace Microsoft.ML
     public static class TextLoaderSaverCatalog
     {
         /// <summary>
-        /// Create a text reader <see cref="TextLoader"/>.
+        /// Create a text loader <see cref="TextLoader"/>.
         /// </summary>
         /// <param name="catalog">The <see cref="DataOperations"/> catalog.</param>
```

Comment:
The doc string to `catalog` is not clear. Could you make a better one? It doesn't describe its relation to other operators and the text loader being created.

Reply:
This is actually an extension method to MLContext. The doc string for catalog is not visible to the user.